Documentation/ApiOverview/Events/Events/Core/Domain/_RecordCreationEvent/_Coordinates.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Model;
+
+class Coordinates
+{
+    public float $latitude = 0.0;
+    public float $longitude = 0.0;
+
+    /**
+     * @param mixed $value - Accepts a string (e.g., "12.34,56.78")
+     */
+    public function __construct(mixed $value)
```

Comment:
Should we use string here to simplify the example. Especially, because no handling for other types are implemented.

Reply:
$event->getProperty() returns mixed. So the handling of types should be delegated to the constructor instead of causing a type error... Usually I would now log or throw an error if an unexpected type is received. But I didnt want to blow up the example...

Reply:
ACK